cmd/observe/observe_filter.go

```diff
@@ -159,6 +160,35 @@ func (of *observeFilter) checkConflict(t *filterTracker, name, val string) error
 	return nil
 }
 
+func (of *observeFilter) ParseTCPFlags(val string, flags *pb.TCPFlags) error {
```

Comment:
nit: I see no use for the receiver, so I think this function could simply be written as:
```golang
func parseTCPFlags(val string) (*pb.TCPFlags, error)
```